platform-emf/src/main/java/com/softicar/platform/emf/data/table/EmfDataTablePath.java

```diff
@@ -0,0 +1,132 @@
+package com.softicar.platform.emf.data.table;
+
+import com.softicar.platform.common.core.i18n.IDisplayString;
+import com.softicar.platform.common.core.utils.CastUtils;
+import com.softicar.platform.common.string.Imploder;
+import com.softicar.platform.common.string.hash.Hash;
+import com.softicar.platform.dom.elements.popup.DomPopup;
+import com.softicar.platform.dom.elements.popup.compositor.CurrentDomPopupCompositor;
+import com.softicar.platform.dom.elements.tab.DomTabBar;
+import com.softicar.platform.dom.node.IDomNode;
+import com.softicar.platform.emf.form.popup.EmfFormPopup;
+import com.softicar.platform.emf.page.IEmfPageContentElement;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Objects;
+import java.util.Optional;
+
+public class EmfDataTablePath {
```

Comment:
I am completely unsure how this will work in practice.

Reply:
In which regard?

Reply:
I just cannot see the implications and if we will not run into new problems.